packages/sdk/src/sdk/sdk.ts

```diff
@@ -505,6 +506,12 @@ const initializeApis = ({
     middleware
   })
 
+  const remixContests = new RemixContestApi(
```

Comment:
Related to other comment above: I think this makes sense as a single EventsApi that has functions for creating/manipulating remix contests.